sklearn/tests/test_naive_bayes.py

```diff
@@ -688,7 +688,6 @@ def test_categoricalnb():
     clf.fit(X3, y3)
     assert_array_equal(clf.n_categories_, np.array([3, 6]))
 
-    # Check error is raised for X with negative entries
```

Comment:
I do not think this is related to this PR. We can revert this.

Reply:
Ops, sorry. I pushed the wrong modification. I reverted and added the test.